src/components/FiltersPanel/FiltersPanel.tsx

```diff
@@ -92,68 +59,15 @@ export const FiltersPanel: FC<{
     });
 
     const [layout] = useState<LayoutType>(layoutType.table);
-    const filterTriggerRef = useRef<HTMLButtonElement>(null);
-    const [filterVisible, setFilterVisible] = useState(false);
-    const [ownersQuery, setOwnersQuery] = useState('');
-    const [issuersQuery, setIssuersQuery] = useState('');
-    const [participantsQuery, setParticipantsQuery] = useState('');
-    const [projectsQuery, setProjectsQuery] = useState('');
-    const [tagsQuery, setTagsQuery] = useState('');
     const [filterQuery, setFilterQuery] = useState<Partial<FilterQueryState> | undefined>(queryFilterState);
+    const filterQueryRef = useLatest(filterQuery);
 
     useEffect(() => {
         setFilterQuery(queryState);
     }, [queryState]);
 
-    const { data: owners = [] } = trpc.user.suggestions.useQuery(
-        {
-            query: ownersQuery,
-            include: queryState?.owner,
-            take: filtersTakeCount,
-        },
-        useQueryOptions,
-    );
-
-    const { data: issuers = [] } = trpc.user.suggestions.useQuery(
-        {
-            query: issuersQuery,
-            include: queryState?.issuer,
-            take: filtersTakeCount,
-        },
-        useQueryOptions,
-    );
-
-    const { data: participants = [] } = trpc.user.suggestions.useQuery(
-        {
-            query: participantsQuery,
-            include: queryState?.participant,
-            take: filtersTakeCount,
-        },
-        useQueryOptions,
-    );
-
-    const { data: projects = [] } = trpc.project.suggestions.useQuery(
-        {
-            query: projectsQuery,
-            include: queryState?.project,
-            take: filtersTakeCount,
-        },
-        useQueryOptions,
-    );
-
-    const { data: tags = [] } = trpc.tag.suggestions.useQuery(
-        {
-            query: tagsQuery,
-            include: queryState?.tag,
-            take: filtersTakeCount,
-        },
-        useQueryOptions,
-    );
-    const { data: states = [] } = trpc.state.all.useQuery();
-    const { data: priorities = [] } = trpc.priority.getAll.useQuery();
-
     const setPartialQueryByKey = useCallback(<K extends keyof QueryState>(key: K) => {
```

Comment:
Assume, we can remove this logic and push values directly to router by batchQueryState if we update state per every filter